File: array/leetcode_1480_Running_)Sum_of_1d_Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
 var runningSum = function(nums) {
    
    var ans = [];
    ans.push(nums[0]);
    l = nums.length;

    for(var i=1;i<l;i++){
        ans.push(ans[i-1]+nums[i])
    }
        
    return ans;
    
};

File: array/leetcode_1480_Running_)Sum_of_1d_Array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        
        ans = []
        ans.append(nums[0])
        l = len(nums)
        
        for i in range(1,l,1):
            ans.append(ans[i-1]+nums[i])
            
        return ans